remarkable-review/The End of Paper.md

```diff
@@ -0,0 +1,101 @@
+
+## A dalliance with digital
+As wonderful as paper had been to me, as a digital native, I was constantly aware of the tradeoff I was making: digital has its advantages.
+
+I got my first job as a software developer in 2003. Which means I type a lot. My keyboard is often the most expedient tool for making quick notes.
+
+It began with simple "scratch notes", outlining how I was planning to tackle a problem or something I wanted to remember for later in the day. Eventually, I found myself needing to recall something from a previous day or share something with a colleague.
+
+In 2011, I decided my digital note-taking needed a system of it's own. I signed up for [Simplenote](https://simplenote.com/) and began accumulating notes. Over the next decade, I migrated to NVAlt and then Bear, bringing my growing collection of notes with me each time. In tandem with my paper note-taking practice, I iterated on my digital note-taking process.
+
+Today, I have 706 notes, including this article. I've been bitten by Zettelkasten bug. If you haven't heard of that particular insect, it's mostly harmless: it causes you to lose track of time whenever you have an idea.
+
+If you're really into organizational systems, I recorded a [livestream](livestream) where I whiteboarded my reading, writing, and note-taking systems.
+
+My digital note repository does have advantages over my multitudinous Moleskines.
+
+**Search.** I can instantly summon all the notes I've written in the past decade containing a search term. This is tremendously helpful when I know what I'm looking for and I want to go there right away.
+
+Conversely, my paper notebooks are organized in a linear fashion, with each day's consecutive note following the previous one. I attempted to use indexes and break up sections by topic, but it never stuck. Currently, I just have tabs that allow me to quickly jump to a specific date.
+
+<picture of my paper notebooks with date tabs>
+
+**Organization.** In addition to search, digital notes can be organized in a variety of non-linear ways, including tags and folders. Tags and folders can be helpful, but surprisingly it hasn't proved as valuable as I expected. I sometimes use a specific tag to focus in on a set of notes and "hide" my other notes, but for some reason a hierarchical tagging system hasn't really helped me.
+
+The paper equivalent would be if paper pages could be removed from their binding and reshuffled.
+
+<TODO: Cut this one?>
+
+**Sharing and collaboration.** I love the ease with which I can share a digital note with coworkers or migrate them to a collaborative document like Google Docs or Notion. Paper is still hard to share. Even with ubiquitous smartphones with cameras, wifi, and apps to quickly "scan" a document (I use the excellent [Office Lens](https://apps.apple.com/us/app/microsoft-office-lens-pdf-scan/id975925059)), it's still not as easy to share paper as it is digital text.
```

Comment:
"ubiquitous smartphones with cameras" -> "ubiquitous smartphone cameras" to cut out saying "with" back to back